src/utils/Constants.tsx

```diff
@@ -0,0 +1,48 @@
+import CottageOutlinedIcon from '@mui/icons-material/CottageOutlined'
+import ContentPasteOutlinedIcon from '@mui/icons-material/ContentPasteOutlined'
+import TuneOutlinedIcon from '@mui/icons-material/TuneOutlined'
+import Person2OutlinedIcon from '@mui/icons-material/Person2Outlined'
+import UsbOutlinedIcon from '@mui/icons-material/UsbOutlined'
+import VerifiedUserOutlinedIcon from '@mui/icons-material/VerifiedUserOutlined'
+import SummarizeOutlinedIcon from '@mui/icons-material/SummarizeOutlined'
```

Comment:
instead of multiple line imports. Modify it into single line

example
`
import { DataGrid, GridColDef, GridValueGetterParams } from '@mui/x-data-grid';
`

Reply:
ok,done